src/internal/Subject.ts

```diff
@@ -41,94 +42,91 @@ export class Subject<T> extends Observable<T> implements SubscriptionLike {
 
   lift<R>(operator: Operator<T, R>): Observable<R> {
     const subject = new AnonymousSubject(this, this);
-    subject.operator = <any>operator;
-    return <any>subject;
+    subject.operator = operator as any;
+    return subject as any;
   }
 
-  next(value: T) {
+  protected throwIfClosed() {
```

Comment:
issue: I think this protected method should have a leading underscore, too. I would not want this to be seen as part of the public API. I'd like us to reserve the right to remove it and replace it if there is an alternative that offers better perf - as this is, AFAICT, only a code-size-related change.

Reply:
Also, whilst in `Subject.ts`, it really irks me that each of these members is implicitly public:

https://github.com/ReactiveX/rxjs/blob/480b7c83c733af35aaa402735bb832652cb9e45d/src/internal/Subject.ts#L17-L25

How about making only `closed` (explicitly) public (for the class's implementation of `SubscriptionLike`) and each of the others either protected or private as appropriate?